modules/API/API.js

```diff
@@ -492,6 +492,10 @@ function initCommands() {
         },
         'set-subtitles': (enabled, displaySubtitles, language) => {
             APP.store.dispatch(setRequestingSubtitles(enabled, displaySubtitles, language));
+        
+            if (enabled) {
+                APP.store.dispatch({ type: 'START_TRANSCRIPTION', language });
```

Comment:
Why is this needed? You define a new action(startTranscription) and that action is nowhere handled.